src/components/WwwFrame/TheAppInstallBanner.vue

```diff
@@ -8,6 +8,47 @@
 
 <script>
 export default {
+	metaInfo() {
+		return {
+			meta: this.isAppInstallPage ? [
+				// Apple specific meta tag to show native app install banner
+				// This is all that we need to do for iOS
+				{
+					name: 'apple-itunes-app',
+					content: 'app-id=1453093374',
+				}
+			] : []
+		};
+	},
+	computed: {
+		isAppInstallPage() {
+			const route = this.$route;
+			const baseUrlsToInclude = [
+				'start',
+				'portfolio',
```

Comment:
Realizing that several of these urls (and their children) are in the old stack.

Reply:
yes and login + register in a different repo as well. I know this component will have to be replicated somewhat in the old stack but may or may not be reusable in the Auth0 repo. Not sure if we're using vue-meta there.